trinity/protocol/les/trackers.py

```diff
@@ -37,3 +39,21 @@ def _get_result_size(self, result: Tuple[BlockHeader, ...]) -> int:
 
     def _get_result_item_count(self, result: Tuple[BlockHeader, ...]) -> int:
         return len(result)
+
+
+BaseGetBlockBodiesTracker = BasePerformanceTracker[
+    GetBlockBodiesRequest,
+    Tuple[BlockBody, ...],
+]
+
+
+# Q: Where can I find the signature for this class?
```

Comment:
I don't know what you mean by *signature*

Reply:
I'm not sure either, feel free to hop on chat, since it may require some back-and-forth to resolve.

Reply:
ah, i just meant the input arguments and the expected return